File: platform-101/02-review-code.ts
import "dotenv/config";
import Anthropic from "@anthropic-ai/sdk";

const client = new Anthropic();

const buggyCode = `
function add(a, b) {
  return a + b;
}
`;

const response = await client.messages.create({
  model: "claude-opus-4-8",
  max_tokens: 1024,
  system: "You are a terse senior code reviewer. Give feedback in one paragraph.",
  messages: [
    { role: "user", content: `Review this code:\n${buggyCode}` },
  ],
});

for (const block of response.content) {
  if (block.type === "text") {
    console.log(block.text);
  }
}

File: platform-101-python/02_review_code.py
from dotenv import load_dotenv
load_dotenv()

import anthropic

client = anthropic.Anthropic()

buggy_code = """
function add(a, b) {
  return a + b;
}
"""

response = client.messages.create(
    model="claude-opus-4-8",
    max_tokens=1024,
    system="You are a terse senior code reviewer. Give feedback in one paragraph.",
    messages=[
        {"role": "user", "content": f"Review this code:\n{buggy_code}"},
    ],
)

for block in response.content:
    if block.type == "text":
        print(block.text)
